lefthook.yml

```
@@ -0,0 +1,7 @@
+pre-commit:
+  parallel: true
+  commands:
+    lint:
+      run: pnpm lint:fix
+    prettier:
+      run: pnpm prettier
```

Make to only run for staged files.
```suggestion
        eslint:
      glob: "*.{vue,ts}" 
      run: pnpm dlx eslint --fix {staged_files}  # Lint staged files and auto-fix if possible

    prettier:
      glob: "*.{vue,ts}" 
      run: pnpm dlx prettier --write {staged_files} # Format staged files in place

```